packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/location.dart

```diff
@@ -138,3 +142,51 @@ class LatLngBounds {
   @override
   int get hashCode => Object.hash(southwest, northeast);
 }
+
+/// A data point entry for a heatmap.
```

Comment:
If it's only for heatmaps, maybe this should be in heatmap.dart.

Reply:
Done